scripts/check-json-for-parsing-errors.rb

```diff
@@ -0,0 +1,22 @@
+# Check JSON for parsing errors
+#
+# Given a few file paths, this script will check if there have been any Excel
+# parsing errors, because of problems in the source excel file.
+# To run on a whole directory of JSON files:
+# $ ruby scripts/check-json-for-parsing-errors.rb ../ifsc-api/data/*.json
+
+require 'json'
```

Comment:
We already have tests in the repo. I'd have preferred to write a simple test for this, rather than another script.